apps/mobile/src/app/assets/index.ts

```diff
@@ -1,6 +1,7 @@
 import logo from './images/logo.png';
 import textLogo from './images/text-logo.png';
 import auctionHeart from './images/auction-heart.png';
+import heartCredit from './images/heart-credit.png';
```

Comment:
- [ ] You should use the icon component/atom. 

You can use the icon properties to change the size and colour instead of adding new Image. 

Example usage: 

`   <Icon name={IconType.favorite} size={200} color='#960e8a' />`